### PR TITLE
Improve formatting and language used in UG

### PR DESCRIPTION
**Pull request by minosx31**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -309,22 +312,24 @@ Format: `delete INDEX`
...
+- `find Betsy` followed by `delete 1` <br>
+The 1st person in the resulting list of the `find` command will be deleted.
 
-[Return to Top](#table-of-contents)
+[Back to Top ↑](#table-of-contents)
 
 ### Clearing all entries : `clear`
 
-Clears all entries from AIMS.
+Lets you clear all existing entries in AIMS.
```
Comment (minosx31) [inline docs/UserGuide.md:324]: Should we leave this as `clear all entries` or `clear all persons/information`?

Comment (niekis) [reply docs/UserGuide.md:324]: can standardize i guess? is the rest of the UG using persons?

Comment (minosx31) [reply docs/UserGuide.md:324]: yeah iirc mostly using persons

File: docs/UserGuide.md [hunk 2]
```diff
@@ -94,7 +98,7 @@ Here are some symbols used throughout the user guide to inform you of additional
 - Items with `…` after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as <code>&nbsp;</code> (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
-- Parameters can be in any order.<br>
+- Parameters can be given in any order.<br>
```
Comment (rexcyrio) [inline docs/UserGuide.md:101]: I think we can use "you" language here too!

"You can give the parameters in any order"

File: docs/UserGuide.md [hunk 3]
```diff
@@ -156,36 +158,39 @@ The given CSV file must fulfill the following conditions:
...
 
-![demo sheets](images/demoSheets.png)
-- valid CSV file:
+  ![demo sheets](images/demoSheets.png)
 
-![demo CSV](images/demoCsv.png)
+- A valid CSV file:
 
-- `importcsv C:\file\6coy_platoon_four_data.csv`
+  ![demo CSV](images/demoCsv.png)
```
Comment (rexcyrio) [inline docs/UserGuide.md:167]: What's the purpose of showing the user these screenshots? Maybe @niekis can chime in here?

Comment (rexcyrio) [reply docs/UserGuide.md]: Resolved in a later commit

File: docs/UserGuide.md (hunk 1 above)
Comment (rexcyrio) [reply docs/UserGuide.md:359]: I think we can stick with "entries" -- saying that the "persons" are in AIMS and that we clear them from AIMS sounds a bit weird

**Review by rexcyrio — APPROVED** [review]
LGTM :+1:

**Pull request merged**